packages/control-property-editor/src/panels/outline/OutlinePanel.scss

```diff
@@ -0,0 +1,103 @@
+.filter-outline {
+    display: flex;
+    margin: 20px 14px 30px 14px;
+    flex-direction: row;
+    align-items: center;
+    overflow-y: auto;
+}
+
+.funnel-icon {
+    margin-left: 16px;
+    i {
+        line-height: 20px;
+    }
+}
+
+.funnel-call-out-checkbox {
+    padding-bottom: 5px;
+}
+
+.tree-no-control-found {
+    font-style: normal;
+    font-weight: bold;
+    font-size: 24px;
+    line-height: 29px;
+    text-align: center;
+    color: var(--vscode-foreground);
+}
+.tree-modify-search-input {
+    font-style: normal;
+    font-weight: normal;
+    font-size: 13px;
+    line-height: 18px;
+    text-align: center;
+    padding-top: 20px;
+    color: var(--vscode-foreground);
+}
+.tree-search-icon {
+    text-align: center;
+}
+
+.tree-cell {
+    display: flex;
+    font-size: 13px;
+    font-weight: normal;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    height: 18px;
+    padding: 5px 0;
+    min-width: 0px;
+    align-items: center;
+}
+
+.tree-row {
+    display: flex;
+    justify-content: space-between;
+    &:hover {
+        cursor: pointer;
+        background-color: var(--vscode-dropdown-background);
+        color: var(--vscode-dropdown-foreground);
+        // color: var(--vscode-list-activeSelectionForeground);
+        // background-color: var(--vscode-list-activeSelectionBackground);
```

Comment:
commented code

Reply:
Removed in [3d6b592](https://github.com/SAP/open-ux-tools/pull/1197/commits/3d6b5925479e8e9232c6e76d611038d40a3aeaea)